src/shared/tpm2-util.c

```diff
@@ -773,7 +775,7 @@ size_t tpm2_tpms_pcr_selection_weight(const TPMS_PCR_SELECTION *s) {
 
         uint32_t mask;
         tpm2_tpms_pcr_selection_to_mask(s, &mask);
-        return (size_t)__builtin_popcount(mask);
+        return popcount(mask);
```

Comment:
this loses the cast to size_t, is that intentional?

Reply:
Yes. I think the case serves no purpose, because `unsigned` is promoted to `size_t` automatically. The cast would only have the effect of hiding a type mismatch, if there was a type mismatch.